app/assets/stylesheets/home.css.scss

```diff
@@ -306,5 +306,13 @@
 footer {
   a img {
     max-height: 80px;
+    max-width: 180px;
   }
 }
+
+#queens_library_logo {
```

Comment:
@daryanypl 'queens_library_logo' style commented?

Reply:
The hash is so we can reference the style by the css element id.
The contents are commented out, because I don't yet have the QPL logo to style.  
I'll take this out, until we get the logo.